Add IL2001-IL2060 trimmer warnings (#25842)
dotnet/docs: docs/core/deploying/trim-warnings/il2016.md

---
title: "IL2016: Could not find event on type in Trimmer data format file"
description: "Learn about trim warning IL2016: DataFormatEventNotFound"
ms.date: 08/25/2021
ms.topic: reference
author: mateoatr
ms.author: matorre
f1_keywords:
  - "IL2016"
---
# IL2016: Could not find event on type in Trimmer data format file

## Cause

> An event specified in a Trimmer data format file could not be found.

Reviewer: "Trimmer data format" does feel right - it took me a while (and lookup in the code) to actually understand what this means. I would simply remove it since linker will actually not produce that text, it will simply say "Could not find event on type" (and it will include the name of the file where the problem is).

Author: Should all references to "trimmer data format" be removed?

Reviewer: I think so

## Rule description

> An event specified in a [Trimmer data format](https://github.com/mono/linker/blob/main/docs/data-formats.md)

Reviewer: ```suggestion
An event specified in a [xml file for the trimmer](https://github.com/mono/linker/blob/main/docs/data-formats.md)
```
